zeroconf/__init__.py

```diff
@@ -540,7 +540,9 @@ def __ne__(self, other: Any) -> bool:
     def __repr__(self) -> str:
         """String representation"""
         try:
-            return self.to_string(str(socket.inet_ntoa(self.address)))
+            return self.to_string(str(
```

Comment:
`str()` isn't necessary here (not sure why it was here), `inet_ntop` already returns `str`

Reply:
Done in 8bf9e3f